.rubocop.yml

```diff
@@ -22,6 +22,8 @@ Metrics/CyclomaticComplexity:
   Enabled: false
 Metrics/MethodLength:
   Enabled: false
+Metrics/ClassLength:
+  Enabled: false
```

Comment:
- [ ] I would not recommend editing the `rubocop.yml` file. Please use the file without editing or updating. You can find this [here](https://github.com/microverseinc/linters-config/blob/master/ruby/.rubocop.yml).

Reply:
Thank you very much @Dipeshtwis , we refactor our code and remove the added linter config line.